versions.json

```diff
@@ -1,7 +1,7 @@
 {
-    "8"  : "8.392.08.1",
-    "11" : "11.0.21.9.1",
-    "17" : "17.0.9.8.1",
-    "21" : "21.0.1.12.1"
+    "8"  : "8.402.07.1",
```

Comment:
This should be `8.402.06.1` since 07 was mac only.

Reply:
fixed